api/SVGAElement.json

```diff
@@ -361,7 +362,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": false
+            "deprecated": true
```

Comment:
FYI, FF was the last thing using this, so even though this is standard it is now not supported anywhere, and hence deprecated.